changelog/fix_an_error_for_internal_affairs_location_line_equality_comparison_inside_block.md

```diff
@@ -0,0 +1 @@
+* [#12012](https://github.com/rubocop/rubocop/pull/12012): Fix an error for `InternalAffairs/LocationLineEqualityComparison` when inside block. ([@ydah][])
```

Comment:
This changelog entry is unnecessary because it is an internal cop.

Reply:
Ah! That's right 😅 I updated this PR 🙏